### PR TITLE
Pre-populate email value in Change your (signup) email form

### PR DESCRIPTION
**Pull request by BryanEliDimas**
Part of openstax/business-intel#851

### CODE REVIEW

File: app/controllers/newflow/login_signup_controller.rb [hunk 1]
```diff
@@ -70,6 +73,10 @@ def confirmation_form
       @email = unverified_user.email_addresses.first.value
     end
 
+    def change_your_email
+      @email = unverified_user.email_addresses.first.value
```
Comment (steveburkett) [inline app/controllers/newflow/login_signup_controller.rb]: can anything in line 77 be nil at any time?  this is assuming that email_addresses is an array and always has something.

Comment (steveburkett) [reply app/controllers/newflow/login_signup_controller.rb]: also, the name of this function seems a bit off from what the code is doing, which is setting email from an unverified user.  should the def be called:    set_email_from_unverified_user?

Comment (BryanEliDimas) [reply app/controllers/newflow/login_signup_controller.rb]: Nope. An unverified user is a user that started but hasn't finished signing up. They must always have one email address. A user `has_many :email_addresses`, so it is indeed always an array.

`change_your_email` is not just a function to set the email. It's actually the name of a controller action which has its corresponding view called `change_your_email.html.erb`.

**Comment by BryanEliDimas** [comment]
Thanks for approving the PR. Addressed your comments. Merged.

**Pull request merged**